src/game/gameState.ts

```diff
@@ -24,5 +24,6 @@ export interface GameState {
   gameMode: GameMode;
   turnDuration: number;
   modelType: ModelType;
+  modelRef?: string;
```

Comment:
I'd probably choose a different name, either `modelURL` or `modelHref`, or something similar. The word "ref" has special meaning in React (https://legacy.reactjs.org/docs/refs-and-the-dom.html), so it would be easy to confuse this (it confused me at first).

Reply:
done